sdk/documenttranslation/Azure.AI.DocumentTranslation/src/TranslationConfiguration.cs

```diff
@@ -9,5 +10,54 @@ namespace Azure.AI.DocumentTranslation
     [CodeGenModel("BatchRequest")]
     public partial class TranslationConfiguration
     {
+        /// <summary>
+        /// Initializes a new instance of TranslationConfiguration.
+        /// </summary>
+        /// <param name="sourceUri">The SAS URL for the source container containing documents to be translated.</param>
+        /// <param name="targetUri">The SAS URL for the source container containing documents to be translated.</param>
+        /// <param name="targetLanguageCode">Language code to translate documents to. For supported languages see
+        /// <a href="https://docs.microsoft.com/azure/cognitive-services/translator/language-support#translate"/>.</param>
+        /// <param name="glossary">Custom translation glossary to be used in the translation operation. For supported file types see
+        /// <see cref="DocumentTranslationClient.GetGlossaryFormatsAsync(System.Threading.CancellationToken)"/>.</param>
+        public TranslationConfiguration(Uri sourceUri, Uri targetUri, string targetLanguageCode, TranslationGlossary glossary = default)
```

Comment:
@maririos should these be named sourceBlobContainerSas and targetBlobContainerSas like we had in the old overload?

Reply:
hmmm i think for now let's leave them generic. From the arch board, other approach is to add a `FromAzureStorage` method where we could use those names, but I don't think it is necessary for now